garrysmod/lua/vgui/dmenuoptioncvar.lua

```diff
@@ -17,12 +16,34 @@ function PANEL:Init()
 
 end
 
+function PANEL:SetConVar( strConVar )
+
+	self.m_strConVar = strConVar
+	self.m_ConVar = GetConVar( strConVar )
+
+end
+
+function PANEL:GetConVar()
+
+	return self.m_strConVar
+
+end

```

Comment:
This can definitely be removed in favor of restoring line 6. `AccessorFunc( PANEL, "m_strConVar", "ConVar" )`

Reply:
While keeping the custom SetCVar right? so it gets overridden?

Reply:
Ye